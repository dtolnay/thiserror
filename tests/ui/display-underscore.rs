use thiserror::Error;

#[derive(Error, Debug)]
#[error("{_}")]
pub struct Error;

fn main() {}
