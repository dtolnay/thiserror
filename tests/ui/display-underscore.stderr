error: invalid format string: invalid argument name `_`
 --> tests/ui/display-underscore.rs:4:11
  |
4 | #[error("{_}")]
  |           ^ invalid argument name in format string
  |
  = note: argument name cannot be a single underscore
